embedding/csharp/Tizen.Flutter.Embedding/Interop/Interop.cs

```diff
@@ -60,28 +61,33 @@ public struct FlutterDesktopEngineProperties
             FlutterDesktopEngine engine);
 
         [DllImport("flutter_tizen.so")]
-        public static extern IntPtr FlutterDesktopGetWindow(
+        public static extern IntPtr FlutterDesktopPluginRegistrarGetNativeWindow(
             FlutterDesktopPluginRegistrar registrar);
 
         [DllImport("flutter_tizen.so")]
-        public static extern void FlutterDesktopNotifyAppControl(
+        public static extern void FlutterDesktopEngineNotifyAppControl(
             FlutterDesktopEngine engine,
             SafeAppControlHandle handle);
 
         [DllImport("flutter_tizen.so")]
-        public static extern void FlutterDesktopNotifyLocaleChange(
+        public static extern void FlutterDesktopEngineNotifyLocaleChange(
             FlutterDesktopEngine engine);
 
         [DllImport("flutter_tizen.so")]
-        public static extern void FlutterDesktopNotifyLowMemoryWarning(
+        public static extern void FlutterDesktopEngineNotifyLowMemoryWarning(
             FlutterDesktopEngine engine);
 
         [DllImport("flutter_tizen.so")]
-        public static extern void FlutterDesktopNotifyAppIsResumed(
+        public static extern void FlutterDesktopEngineNotifyAppIsResumed(
             FlutterDesktopEngine engine);
 
         [DllImport("flutter_tizen.so")]
-        public static extern void FlutterDesktopNotifyAppIsPaused(
+        public static extern void FlutterDesktopEngineNotifyAppIsPaused(
+            FlutterDesktopEngine engine);
+
+        [DllImport("flutter_tizen.so")]
+        public static extern FlutterDesktopView FlutterDesktopViewCreateFromNewWindow(
+            ref FlutterDesktopWindowProperties window_properties,
```

Comment:
Will you add these missing functions later when they become necessary?

- `FlutterDesktopEngineNotifyAppIsInactive`
- `FlutterDesktopEngineNotifyAppIsDetached`
- `FlutterDesktopPluginRegistrarGetNativeWindow`

Reply:
Yes, I'll add them when I need.